quickget

```diff
@@ -76,7 +76,8 @@ function pretty_name() {
     ubuntu-unity)       PRETTY_NAME="Ubuntu Unity";;
     vanillaos)          PRETTY_NAME="Vanilla OS";;
     void)               PRETTY_NAME="Void Linux";;
-    vxlinux)            PRETTY_NAME="VX Linux";;
+    vxlinux)            PRETTY_NAME="VX linux";;
```

Comment:
That should be left as "VX Linux"

https://github.com/dessington/vx-linux/blob/d68b13d0a73afc7b8f3ef9649cfd17668947adca/README.md?plain=1#LL1C1-L1C1